planex/cmd/clone.py

```diff
@@ -19,10 +19,10 @@ def parse_args_or_exit(argv=None):
     Parse command line options
     """
     parser = argparse.ArgumentParser(description='Clone sources')
-    parser.add_argument("-P", "--pins-dir", default="PINS",
-                        help="Directory containing pin overlays")
     parser.add_argument("--jenkins", action="store_true",
                         help="Print Jenkinsfile fragment")
+    parser.add_argument("--skip-base", default=False, action="store_true",
```

Comment:
Yes, we've done this before.  I did it this way here so that the `if` condition below comes out as `if  not skip_base`, which is clearly related to the name of the command-line parameter for someone reading the code.   I'm not sure whether this is more valuable than avoiding negatives in the if statements, though...